### PR TITLE
added: Tagger class

### PR DESCRIPTION
**Pull request by Vaydheesh**
solves https://github.com/apertium/apertium-python/issues/61

- added tests for `Tagger`
- updated `Readme`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,11 +61,32 @@ In [1]: import apertium
 In [2]: apertium.append_pair_path('..')
 ```
 
+### Tagger
+Method 1:  One can create ```Tagger``` objects on which ```tag()``` function can be run.
```
Comment (sushain97) [inline README.md]:
```suggestion
Method 1:  One can create `Tagger` objects on which `tag()` function can be run.
```

File: README.md [hunk 2]
```diff
@@ -61,11 +61,32 @@ In [1]: import apertium
...
+Method 2: Running ```tag()``` directly.
+```python
+In [1]: import apertium
+In [2]: apertium.tag('en', 'cats')
+Out[2]: [cats/cat<n><pl>]
+```
+
 ### Translation
-Performing Translations
+Method 1:  One can create ```Translator``` objects on which ```translate()``` function can be run.
```
Comment (sushain97) [inline README.md]:
```suggestion
Method 1:  One can create `Translator` objects on which `translate()` function can be run.
```

File: README.md [hunk 3]
```diff
@@ -61,11 +61,32 @@ In [1]: import apertium
...
 
+### Tagger
+Method 1:  One can create ```Tagger``` objects on which ```tag()``` function can be run.
+```python
+In [1]: import apertium
+In [2]: tagger = apertium.Tagger('eng')
+In [3]: tagger.tag('cats')
+Out[3]: [cats/cat<n><pl>]
+```
+Method 2: Running ```tag()``` directly.
```
Comment (sushain97) [inline README.md]:
```suggestion
Method 2: Running `tag()` directly.
```

File: README.md (hunk 1 above)
Comment (sushain97) [reply README.md]: This doesn't read well. Something like Create a tagger object and call it's tag method is better. Same below.

File: README.md (hunk 3 above)
Comment (sushain97) [reply README.md]: Call, not run.

File: README.md (hunk 2 above)
Comment (sushain97) [reply README.md]: Same comment as above. I would note somewhere that this approach is more performant for multiple invocations.

**Comment by Vaydheesh** [comment]
oops. I was trying to see its significance, and forgot to revert it back

File: README.md [hunk 4]
```diff
@@ -6,31 +6,31 @@
...
 
 ## About the Exisiting Code Base
-- The exisiting code base has the subprocess implementation of the basic functions of Apertium. 
-- A branch called the ```windows``` has the implementation for the ```windows``` support and will soon be available on master. Detailed instructions can be found [here](https://gist.github.com/arghyatiger/c8aab476022158f4bdb3dbe45308cdb4)
+- The exisiting code base has the subprocess and swig wrapper implementation of the basic functions of Apertium. 
 
-## Major things to do
-- Subprocess implementation of the C++ functions in Apertium. To make the wrapper thinner.
-- Other small issues can be found [here](https://github.com/apertium/apertium-python/issues)
+## ToDo
```
Comment (sushain97) [inline README.md]: I would just remove this entire section.

File: README.md [hunk 5]
```diff
@@ -6,31 +6,31 @@
...
 
 ## Usage of library
 
+- For multiple invocations `Method 1` is more performant, as the dictionary needs to be loaded only once.
+
 ### Analysis
 Performing Morphological Analysis
 
-Method 1: One can create ```Analyzer``` objects on which ```analyze()``` function can be run.
+Method 1: Create a `Analyzer` object and call its `analyze` method.
```
Comment (sushain97) [inline README.md]: an

File: README.md [hunk 6]
```diff
@@ -6,31 +6,31 @@
...
 ```python
 In [1]: import apertium
 In [2]: a = apertium.Analyzer('en')
 In [3]: a.analyze('cats')
 Out[3]: [cats/cat<n><pl>, ./.<sent>]
 ```
-Method 2: Alternatively, the library provides an option to directly run the ```analyze``` method.
+Method 2: Calling `analyze` method directly.
```
Comment (sushain97) [inline README.md]: Inconsistent with wording below.

**Pull request merged**